chain/epoch-manager/src/lib.rs

```diff
@@ -506,19 +507,33 @@ impl EpochManager {
             if ProtocolFeature::ChunkEndorsementsInBlockHeader
                 .enabled(epoch_info.protocol_version())
             {
+                // Compares validator accounts by applying comparators in the following order:
+                // First by online ratio, if equal then by stake, if equal then by account id.
+                let validator_comparator =
+                    |left: &(BigRational, &AccountId), right: &(BigRational, &AccountId)| {
+                        let cmp_ratio = left.0.cmp(&right.0);
+                        if cmp_ratio != Ordering::Equal {
+                            cmp_ratio
+                        } else {
+                            let left_stake =
+                                epoch_info.get_validator_stake(left.1).unwrap_or_default();
```

Comment:
Do we want to silently `unwrap_or_default()` here? If it returns None, something is definitely wrong right? I guess fully panicking is debatable (I would prob lean in favor of that though, because if it's None, I would guess something is definitely going wrong in writing this epoch info for epoch T+2 anyway), but at the least maybe we should log an error?

Reply:
I agree that if it returns None, it must be something badly wrong, because the accounts exercised here come from the loop above (at line 473-503). Thus I will change this to `unwrap_or_else(panic)`.